content/plugin/log/managing.mdx

```diff
@@ -0,0 +1,75 @@
+---
+page_title: 'Home - Plugin Development: Managing Log Output'
+description: >-
+  Filter log outputs by subsystem and verbosity level. Set the log format and output path.
+---
+
+# Managing Log Output
+
+Terraform binary, providers, and provider SDKs can all write logs that provide insight into operations and help you diagnose bugs. You can use environment variables to turn on logging, filter log output, choose the log format, and specify the log output path.
+
+## Enable Logging
+
+Logging is off for all subsystems by default. To enable logging for a Terraform run, set one or more logging environment variables to either [`JSON` format](#log-format) or a [verbosity level](#log-levels). To write:
+
+- **All logs** for Terraform binary, providers, and provider SDKs, set `TF_LOG`. When you report bugs to issue trackers, we recommend setting `TF_LOG=TRACE`.
+- **A subset of logs** (e.g., for one provider), set the only the environment variables for your use case.
+
+The following environment variables turn on logging and can help you filter log output. Some environment variables take precedence over others. For example, `TF_LOG` overrides all other logging variables. If you set `TF_LOG=TRACE` and `TF_LOG_PROVIDER_AZUREM=WARN`, all providers will write logs at the `TRACE` level.
+
+| Variable | Logs Written By | Precedence and Usage |
+| --------------|-------------|-------------|
+| `TF_LOG` | All loggers (Terraform, SDKs, providers) | Overrides all other logging environment variables. |
+| `TF_LOG_CORE` | Terraform binary | Does not include providers. |
+| `TF_LOG_PROVIDER` | All providers and provider SDKs used during the run | Overrides logging environment variables for specific providers (e.g., `TF_LOG_PROVIDER_AWS`) and provider SDKs (e.g., `TF_LOG_PROTO`). Only affects logs from Terraform CLI runs and does not affect logs from provider [acceptance tests](#testing). |
```

Comment:
Okay @bflad I tried to clarify that caveat that we talked about where this only works during CLI runs and not during acceptance testing. I also added more context to the testing section to try to explain a bit more/remind folks of this again. Thoughts? We can edit more if need be!

Reply:
I think this is a great start. Just need to update `TF_LOG_PROTO` to `TF_LOG_SDK` (I think preferable here) or `TF_LOG_SDK_PROTO`

```suggestion
| `TF_LOG_PROVIDER` | All providers and provider SDKs used during the run | Overrides logging environment variables for specific providers (e.g., `TF_LOG_PROVIDER_AWS`) and provider SDKs (e.g., `TF_LOG_SDK`). Only affects logs from Terraform CLI runs and does not affect logs from provider [acceptance tests](#testing). |
```